src/runtime/meter-toast-api.js

```diff
@@ -180,7 +180,10 @@ export class MeterToastApi {
           ViewSubscriptionsResponse,
           this.startNativeFlow_.bind(this)
         );
-        if (!this.deps_.callbacks().hasSubscribeRequestCallback()) {
+        if (
+          !this.deps_.callbacks().hasSubscribeRequestCallback() &&
```

Comment:
Is this still dependent on setting a subscribe request callback?

Reply:
Close duplicate